packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2183,6 +2279,15 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         this.submit(ContainerMessageType.FluidDataStoreOp, envelope, localOpMetadata);
     }
 
+    public submitDataStoreAliasOp(contents: any, localOpMetadata: unknown): void {
```

Comment:
I feel like this method (or rather its content) should move to DataStore.trySetAlias and the later to use existing way to send an op. Maybe other way to say - it's not clear why this layer should care about aliasing details.